docs/building/windows-core.md

```diff
@@ -59,6 +59,12 @@ Import-Module ./build.psm1
 Start-PSBuild
 ```
 
+In case `Start-PSBuild` reports issues with missing PowerShell modules, add the following arguments:
+
+```powershell
+Start-PSBuild -PSModuleRestore -Restore -Clean
```

Comment:
```suggestion
Start-PSBuild -Clean -PSModuleRestore
```
I think `-Restore` is not needed.

Reply:
Sure, no objections from me. Thank you so much!